docs/burial.rst

```diff
@@ -2,20 +2,23 @@ burial
 ======
 
 .. dfhack-tool::
-    :summary: Configures all unowned coffins to allow burial.
-    :tags: unavailable fort productivity buildings
+    :summary: Allows burial in unowned coffins.
+    :tags: fort productivity buildings
+
+Creates a 1x1 tomb zone for each built coffin that doesn't already have one.
 
 Usage
 -----
 
-::
-
-    burial [--pets]
+    ``burial [-d] [-p]``
 
-if the ``--pets`` option is passed, coffins will also allow burial of pets.
+Created tombs allow both dwarves and pets by default. By specifying ``-d`` or
+``-p``, they can be restricted to dwarves or pets, respectively.
 
-Examples
---------
+Options
+-------
 
-``burial --pets``
-    Configures all unowned coffins to allow burial, including pets.
+``-d``
+    Create dwarf-only tombs
+``-p``
+    Create pet-only tombs
```

Comment:
the restriction is actually to "citizens", who are usually dwarves, but not always. Maybe the options should be:
```
``-c``, ``--citizens-only``
``-p``, ``--pets-only``
```

since players may not want *all* coffins to be affected, it would be nice to add a 
``-z``, ``--cur-zlevel`` option to only affect the coffins on the current zlevel.

example of how to process short and long options, and an example of `--cur-zlevel` here: https://github.com/DFHack/scripts/blob/master/drain-aquifer.lua#L41